packages/eckit/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+#
+from spack import *
+from spack.pkg.builtin.eckit import Eckit as SpackEckit
+
+
+class Eckit(SpackEckit):
+
+    git = 'https://github.com/ecmwf/eckit.git'
+    version('1.20.0', branch='1.20.0')
```

Comment:
Have you considered going directly to [1.20.2](https://github.com/ecmwf/eckit/releases/tag/1.20.2)? It seems like it fixes some bugs and keeps compatibility.

I noticed that 1.16.0 is the latest version in spack/develop. Generally I think it's a good idea to add such contributions to spack directly, so everyone can profit and we can profit from the review and testing process in spack. So I created a [PR in spack](https://github.com/spack/spack/pull/33200) to add newer versions.

Reply:
Infero is built with version 1.20.0:
https://github.com/jonasjucker/infero/blob/4c229a16ce75a249c83cbf43e0c953a7a42f2f83/dev/env.sh#L48
And eckit is only used for that atm.